app/frontend/components/common/alert.tsx

```diff
@@ -15,11 +15,12 @@ interface Props {
     | 'info'
     | 'nufi-announcement sidebar'
   children: ComponentChildren
+  alertContentType?: 'nufi-announcement'
 }
 
-const Alert = ({children, alertType = 'success'}: Props) => (
+const Alert = ({children, alertType = 'success', alertContentType}: Props) => (
   <div className={`alert ${alertType}`}>
-    <div className="alert-content">{children}</div>
+    <div className={`alert-content ${alertContentType || ''}`}>{children}</div>
```

Comment:
I'd reuse the alertType here - I don't see in the CSS any pre-existing rules that would cause breaking changes in the existing instances of alert

Reply:
Not really sure if you mean it like this but if I
```
<div className={`alert-content ${alertType || ''}`}>{children}</div>
```
the styles are broken.

I ended up removing it and used CSS selector for that purpose

Reply:
[ded8d40](https://github.com/vacuumlabs/adalite/pull/1284/commits/ded8d40638cd68a1d722c529951f9b6fcc199fe8)

Reply:
Ah I see, probably some css rule concerning the alertType alone broke it, CSS selectors sound good to me :+1: